lua/cfc_prop_restoration/core/cl/cl_init.lua

```diff
@@ -0,0 +1,6 @@
+
+net.Receive( "Restore_AlertReconnectingPlayer", function( len )
+    -- CREATE NOTIFICATION --
+    net.Start( "Restore_RestorePlayerProps" )
+    net.SendToServer()
+end)
```

Comment:
So this requests that props be restored as soon as it gets a ping from the server?

Reply:
It will be eventually hooked up with the notification library the way it is now is a placeholder.